app/assets/javascripts/tests/helpers.js

```diff
@@ -10,6 +10,11 @@ export const mock = name => {
     wasCalled: () => calls1.length === 1,
     wasCalledWith: arg => deepEqual(arg, calls1[calls1.length - 1]),
     wasCalledWith2: arg => deepEqual(arg, calls2[calls2.length - 1]),
-    wasCalledWith3: arg => deepEqual(arg, calls3[calls3.length - 1])
+    wasCalledWith3: arg => deepEqual(arg, calls3[calls3.length - 1]),
+    wasCalledWithParameter: obj => {
```

Comment:
I'd suggest naming it like in rspec (https://relishapp.com/rspec/rspec-mocks/v/3-2/docs/setting-constraints/matching-arguments). Something like `wasCalledWithObjIncluding`. Because parameter seems like it's checking the whole parameter